src/components/network/network-panel.js

```diff
@@ -135,8 +148,21 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.source.label);
```

Comment:
Are you sure?

Renderer has a bit unclear typing on its nodes_ and edges_ for rendering purposes. It would be better if we type annotate nodes_ and edges_. Meanwhile this.data.network.nodes/edges are genotet.networkNode's and genotet.networkEdge's.

Here if you pass in genotet.networkEdge then edge.source would be string referring to the source node id. Then you can use this id to get node label to show.